addons/editor-number-arrow-keys/addon.json

```
@@ -1,6 +1,6 @@
 {
   "name": "Arrow key incrementation in editor",
-  "description": "Use the up and down arrow keys to increase or decrease number values in block inputs.",
+  "description": "Use the up and down arrow keys to increase or decrease number values in numeric input fields, such as the input in the \"wait () seconds\" block or the \"size\" field in the sprite properties.",
```

I would suggest less specific examples:
```suggestion
  "description": "Use the up and down arrow keys to increase or decrease number values in numeric input fields such as those in blocks or the sprite properties area.",
```
Not sure if this suggestion is ideal, but to me it feels like the examples in the current description are a little too "specific" and don't cover every input field they should or something? To me, it should be clear that number inputs in _all_ blocks can be adjusted, and that _all_ numerical sprite properties can be adjusted too.